app/models/form/subsection.rb

```diff
@@ -31,7 +31,7 @@ def status(case_log)
     end
 
     qs = applicable_questions(case_log)
-    return :not_started if qs.all? { |question| case_log[question.id].blank? }
+    return :not_started if qs.all? { |question| case_log[question.id].blank? || question.read_only? }
```

Comment:
This is fine because we're now setting the derived fields before validation so it _should_ always be there right?